src/server/main_service.cc

```diff
@@ -103,6 +103,11 @@ ABSL_FLAG(double, oom_deny_ratio, 1.1,
           "commands with flag denyoom will return OOM when the ratio between maxmemory and used "
           "memory is above this value");
 
+ABSL_FLAG(double, rss_oom_deny_ratio, 1.1,
```

Comment:
@romange do you think this should be enabled by default or disabled by default?

Reply:
enabled but maybe increase the default ratio to `1.25` because it affects `maxmemory` semantics

Reply:
we also need to provide explanation in release notes

Reply:
what if one upgrades to new version and does not use admin port. if he hits the rss limit he will not be able to even connect to the server as we will reject new connections

Reply:
hmm, you are right but how we introduce this behavior then? I do think that some sane multiplier is needed. This can be eve 2,3 but if RSS jumps by a huge factor - it's not ok.

Reply:
maybe we should reject new connections only if admin port is set?

Reply:
sounds good